pkg/broker/ipsec_psk.go

```diff
@@ -5,8 +5,11 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
 )
 
+const IPSECPSKSecretName = "submariner-ipsec-psk"
```

Comment:
Can we avoid exporting this? Or is it needed for the test?

Reply:
I think we don't even need it for the test, we can hide it, yes. Good point.

I also took the chance to camelCase psk_secret_data , even if it's not part of this patch specifically.